deploy/ocp/manifests/0.10.0/0000_50_olm_10-operatorgroup.crd.yaml

```diff
@@ -93,7 +93,6 @@ spec:
                 type: string
               type: array
           required:
-          - namespaces
```

Comment:
I don't think you want to modify the versioned directories as they have already been released.